Android/GoogleDagger/source/Dagger/Dagger.csproj

```
@@ -12,19 +12,20 @@
     <RootNamespace>Annotations</RootNamespace>
     <AndroidClassParser>class-parse</AndroidClassParser>
     <AndroidCodegenTarget>XAJavaInterop1</AndroidCodegenTarget>
+    <NoWarn>CS0108,CS0109,CS0114,CS0618</NoWarn>
   </PropertyGroup>
 
   <PropertyGroup>
     <PackageId>Xamarin.Google.Dagger</PackageId>
     <Title>Google Dagger for Xamarin.Android</Title>
-    <PackageDescription>Xamarin.Android bindings for Google Dagger</PackageDescription>
+    <Summary>Xamarin.Android bindings for Google Dagger</Summary>
     <Authors>Microsoft</Authors>
     <Owners>Microsoft</Owners>
     <Copyright>© Microsoft Corporation. All rights reserved.</Copyright>
     <PackageProjectUrl>https://go.microsoft.com/fwlink/?linkid=2106537</PackageProjectUrl>
     <PackageLicenseUrl>https://go.microsoft.com/fwlink/?linkid=2106538</PackageLicenseUrl>
     <PackageRequireLicenseAcceptance>true</PackageRequireLicenseAcceptance>
-    <PackageVersion>2.25.2</PackageVersion>
+    <PackageVersion>2.13</PackageVersion>
```

This is usually injected when building. Maybe revert this:
```suggestion
    <PackageVersion>2.15.2</PackageVersion>
```

---

Change to new patch version: `2.25.2.1`